test/FacilitatorStart/getConfig.test.ts

```diff
@@ -0,0 +1,410 @@
+import * as sinon from 'sinon';
+import { assert } from 'chai';
+import { Config, FacilitatorConfig } from '../../src/Config';
+import MosaicConfig from '../../src/MosaicConfig';
+import SpyAssert from '../test_utils/SpyAssert';
+import FacilitatorStart from '../../src/FacilitatorOptionParser/FacilitatorStart';
+
+describe('FacilitatorOptionParser.getConfig()', () => {
+  const originChain = '2';
+  const auxChain = '3';
+  const facilitatorConfigPath = './facilitator-config.json';
+  const mosaicConfigPath = './test/mosaic-config.json';
+
+  function spyFacilitatorFromFile(fcConfig: any): any {
+    const spy = sinon.stub(
+      FacilitatorConfig,
+      'fromFile',
+    ).callsFake(
+      sinon.fake.returns(fcConfig),
+    );
+    return spy;
+  }
+
+  function spyFacilitatorFromChain(fcConfig: any): any {
+    const spy = sinon.stub(
+      FacilitatorConfig,
+      'fromChain',
+    ).callsFake(
+      sinon.fake.returns(fcConfig),
+    );
+    return spy;
+  }
+
+  function spyMosaicFromChain(mosaicConfig: any): any {
+    const spy = sinon.stub(
+      MosaicConfig,
+      'fromChain',
+    ).callsFake(
+      sinon.fake.returns(mosaicConfig),
+    );
+    return spy;
+  }
+
+  function spyMosaicfromFile(mosaicConfig: MosaicConfig): any {
+    const spy = sinon.stub(
+      MosaicConfig,
+      'fromFile',
+    ).callsFake(
+      sinon.fake.returns(mosaicConfig),
+    );
+    return spy;
+  }
+
+  function spyConfigFromPath(mosaic: any, facilitator: any): any {
+    const spy = sinon.stub(
+      Config,
+      'fromFile',
+    ).callsFake(
+      sinon.fake.returns(new Config(mosaic, facilitator)),
+    );
+    return spy;
+  }
+
+  it('should fail when origin chain is provided but aux chain id is undefined', () => {
+    const fs: FacilitatorStart = new FacilitatorStart(
+      originChain,
+      undefined as any,
+      mosaicConfigPath,
+      facilitatorConfigPath,
+    );
+
+    assert.throws(
+      () => fs.getConfig(),
+      'both origin_chain and aux_chain_id is required',
+    );
+  });
+
+  it('should fail when origin chain is provided but aux chain id is blank', () => {
+    const fs: FacilitatorStart = new FacilitatorStart(
+      originChain,
+      '',
+      mosaicConfigPath,
+      facilitatorConfigPath,
+    );
+
+    assert.throws(
+      () => fs.getConfig(),
+      'both origin_chain and aux_chain_id is required',
+    );
+  });
+
+  it('should fail when aux chain id is provided but origin chain is undefined', () => {
+    const fs: FacilitatorStart = new FacilitatorStart(
+      undefined as any,
+      auxChain,
+      mosaicConfigPath,
+      facilitatorConfigPath,
+    );
+
+
+    assert.throws(
+      () => fs.getConfig(),
+      'both origin_chain and aux_chain_id is required',
+    );
+  });
+
+  it('should fail when aux chain id is provided but origin chain is blank', () => {
```

Comment:
```suggestion
  it('should fail when aux chain id is an empty string', () => {
```

Similarly for other tests.